lecture-notes/1-cli.md

```diff
@@ -871,12 +871,12 @@ See also the `screen` command, which is similar.
 
 #### `vi`
 `vi` is a POSIX-specified text editor that is available on almost every system
-you will use. Unlike text editors such as Notepad and Kate, is a *modal*
+you will use. Unlike text editors such as Notepad and Kate, `vi` is a *modal*
```

Comment:
Can you style this as "Vi" instead of `vi`, like we do for other mentions of the name later?

Reply:
Done!